src/components/node/Node.jsx

```diff
@@ -94,16 +95,16 @@ export default class Node extends React.Component {
             opacity: this.props.opacity,
         };
 
-        const size = this.props.size;
+        let size = this.props.size;
 
         let gtx = this.props.cx,
             gty = this.props.cy,
             label = null,
             node = null;
 
         if (this.props.svg || this.props.viewGenerator) {
-            const height = size / 10;
-            const width = size / 10;
+            const height = typeof size === "object" ? size.height / 10 : size / 10;
```

Comment:
Please store in a variable the result of `typeof size === "object"` and use it in the 3 places where we're performing the same check